src/js/components/Button/doc.js

```diff
@@ -62,6 +62,11 @@ with plain Buttons.`,
     href: PropTypes.string.description(
       'If specified, the button will behave like an anchor tag.',
     ),
+    target: PropTypes.oneOf(['_self', '_blank', '_parent', '_top'])
+      .description(
+        `Specifies where to display the URL defined in the href property.`,
+      )
+      .defaultValue('_self'),
```

Comment:
I thought there was no default value? In other words, if I inspect the DOM, it doesn't have a `target` attribute set.

Reply:
I didn't realize that's what the `.defaultValue` in this file means.
I thought it was just what behavior will apply if we don't specify anything.
Will change, squash & re-push :+1: